starter-code/src/app/components/my-home-component/my-home-component.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="container">
+  <div *ngFor="let movie of movieList" class="card">
+    <h2>{{movie.title}}</h2>
+    <p>Room {{movie.room}}</p>
+    <a [routerLink]="['/movie',movie]"><img [src]="[movie.poster]"></a>
```

Comment:
`[routerLink]="['/movie',movie]"` you should have the `movie.id` instead of the whole object.